packages/apps/graph/generated-schema.graphql

```diff
@@ -47,6 +47,41 @@ type BlockTransactionsConnectionEdge {
   node: Transaction!
 }
 
+"""A transaction cmd."""
+type Cmd {
```

Comment:
If I remember correctly, we wanted to already start using fully written out names, since that's where the domain will shift to in the future.

cmd -> command
exec -> execution
cont -> continuation

But please confirm @alber70g

Reply:
Yes, lets do that. 👍